osbuild-composer.spec

```diff
@@ -183,13 +184,13 @@ done
 
 %else
 # All other RHEL versions support building for the same version
-for REPO_FILE in $(ls repositories/rhel-%{rhel}* | grep -v 'no-aux-key'); do
+for REPO_FILE in $(ls vendor/github.com/osbuild/images/data/repositories/rhel-%{rhel}* | grep -v 'no-aux-key'); do
```

Comment:
Calling `grep` is no longer needed since there are no `-no-aux-key` repos in `images, right?